recipes/buildbot-console-view/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "buildbot-console-view" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "2a6370db2141aebd7466fee390824fb7190d1e7122d6ebb844690cde141ca2f8"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

The arguments are no longer required.
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```

---

Removing them then. @xhochy would you like to be a maintainer?

---

Sure, I can help with them

---

Me too. Ping me when it is ready and I'm going to merge it